sdk/core/Azure.Core/src/Shared/ClientDiagnostics.cs

```diff
@@ -30,8 +30,10 @@ public ClientDiagnostics(ClientOptions options)
         }
 
         public ClientDiagnostics(string optionsNamespace, string? providerNamespace, DiagnosticsOptions diagnosticsOptions)
-            : base(optionsNamespace, providerNamespace, diagnosticsOptions.IsDistributedTracingEnabled)
+            : base(optionsNamespace, providerNamespace, diagnosticsOptions.IsDistributedTracingEnabled, true)
         {
+            // TODO: use DiagnosticsOptions.IsNestingClientSpansEnabled  here after new Azure.Core ships:
```

Comment:
I think it would be best to update dependent projects to use a project reference, so we can see and test the changes end to end before merging them.  This is a standard Azure.Core procedure -- let us know if you need help with this.

Reply:
discussed offline: it'd require switching everything to ProjectReference and decided not to do it.